test/visual/drag-states.test.ts

```diff
@@ -173,3 +173,20 @@ test(
     expect(await page.fullPageScreenshot()).toMatchImageSnapshot();
   })
 );
+
+test(
+  "items layout is updated if columns change while active transition",
+  setupTest("/index.html#/dnd/update-layout-test", DndPageObject, async (page) => {
```

Comment:
Visual regression tests are not build blocking, so you need to move it to functional folder.

Reply:
Makes sense, will do